src/World.cpp

```diff
@@ -3051,7 +3051,11 @@ void cWorld::QueueTask(cTaskPtr a_Task)
 }
 
 
-
+void cWorld::ScheduleTask(int a_DelayTicks, std::function<void (cWorld&)> a_Func)
+{
+	cTaskLambda task = cTaskLambda(a_Func);
```

Comment:
I did some more digging around, it looks like the actual implementation of SendSound uses an int, instead of double, and almost every upstream caller is casting int to double. It would make sense to refactor all the sound calls in between to accept int instead of double, but I didn't want to make this pull request about refactoring the sound system.